core/base/inc/TROOT.h

```diff
@@ -278,6 +280,7 @@ friend TROOT *ROOT::Internal::GetROOT2();
    Int_t             GetNtypes() const { return fTypes->GetSize(); }
    TFolder          *GetRootFolder() const { return fRootFolder; }
    TProcessUUID     *GetUUIDs() const { return fUUIDs; }
+   TString           GetWeb() const { return fWeb; }
```

Comment:
Can we use Set/GetWebDisplay since TROOT is quite generic ....

Reply:
I am even wondering if 'Web' is the best description of the feature ...

Reply:
> I am even wondering if 'Web' is the best description of the feature ...

Here 'Web' corresponds to web browsers.

Reply:
> Can we use Set/GetWebDisplay since TROOT is quite generic ....

WebDisplay probably better name.

Reply:
Good point.